scripts/travis/travis_wait.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+#
+# Copyright (c) 2016 Travis CI GmbH <contact@travis-ci.org>
+#
+# SPDX-License-Identifier: MIT
+#
+# Based on the travis-ci/travis-build
```

Comment:
I never knew about this. Never really thought about how the YML is actually executed. A link to the actual source files may be useful for future debugging:
https://github.com/travis-ci/travis-build/blob/d8f183b29700c3b98b6995ecc4b463f3c3bc552f/lib/travis/build/bash/travis_jigger.bash
https://github.com/travis-ci/travis-build/blob/d8f183b29700c3b98b6995ecc4b463f3c3bc552f/lib/travis/build/bash/travis_wait.bash